examples/index.json

```diff
@@ -272,7 +272,14 @@
                 "verifyBuild": true,
                 "verifyOutput": true,
                 "isLearnByExample": true
-            }
+            },
+            {
```

Comment:
Not the correct category IMO. We'll probably need a separate category after records.